Userland/Applications/CertificateSettings/main.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2023, Fabian Dellwing <fabian@dellwing.net>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "CertificateStore.h"
+
+#include <LibCore/StandardPaths.h>
+#include <LibCore/System.h>
+#include <LibGUI/Application.h>
+#include <LibGUI/Icon.h>
+#include <LibGUI/SettingsWindow.h>
+#include <LibMain/Main.h>
+#include <unistd.h>
+
+ErrorOr<int> serenity_main(Main::Arguments args)
+{
+    TRY(Core::System::pledge("stdio rpath wpath cpath recvfd sendfd unix"));
+
+    auto app = TRY(GUI::Application::try_create(args));
+
+    TRY(Core::System::unveil(TRY(String::formatted("{}/.config/certs.pem", Core::StandardPaths::home_directory())), "rwc"_short_string));
+    TRY(Core::System::unveil("/tmp/session/%sid/portal/filesystemaccess", "rw"));
+    TRY(Core::System::unveil("/etc/cacert.pem", "r"));
+    TRY(Core::System::unveil("/etc/timezone", "r"));
+    TRY(Core::System::unveil("/res", "r"));
+    TRY(Core::System::unveil(nullptr, nullptr));
+
+    auto app_icon = GUI::Icon::default_icon("certificate"sv);
+    auto window = TRY(GUI::SettingsWindow::create("Certificates", GUI::SettingsWindow::ShowDefaultsButton::No));
+    auto cert_store_widget = TRY(window->add_tab<CertificateSettings::CertificateStoreWidget>(TRY("Certificate Store"_string), "certificate"sv));
```

Comment:
i guess thatʼs something which is called “Trust Store” in other places? thereʼs also a group which exactly describes the content, i.e. “Trusted Root Certificate Authorities”, which is also nice (but iʼm not sure how much it duplicates the content)…

not sure what is the overall design for this settings window. maybe we should remove the group and name the tab that way, or “Trusted Root CAs” for short and keep other certificate/key stores available here…? 🤔

nice job here as a whole—just rough ideas and more of a food for thought for the future to avoid needless work. 🙇🏻‍♂️

Reply:
I did choose the name "Certificate Store" because I would like to manage client certificates here in the future, too.

The group box inside than further describes the current thing as `Trusted Root Certification Authorities`.